### ISSUE TITLE
Request (if possible) | Umbraco (v10) docs as PDF

### ISSUE THREAD

**Issue by jacksorjacksor** [umbraco/UmbracoDocs#5347]
## What type of issue is it? (Choose one - delete the others)
Discussion

## What article/section is this about?
All of it!

## Describe the issue
As the documentation is hosted in GitBook I was hoping it would be possible to export the docs as .PDF [as described here](https://docs.gitbook.com/publishing/share/pdf-export) - however, that does not seem to be the case. I would personally love to have a copy of the docs as a .PDF to read over, so if that is something which can be provided I would be grateful, and could imagine others might feel the same.

If it's not possible then no worries at all!

Cheers!

Rich

**Comment by alina-tincas** [comment]
Hi @jacksorjacksor 👋 

Thank you for reaching out! We currently have this feature on each page of our documentation and can be found if you click on the 3 dots next to main heading of each page:
![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/e2fced9b-78fd-47b1-9708-d675a97937a1)

Can you confirm if the above is what you were looking for? 😊

**Comment by jacksorjacksor** [comment]
Morning @alina-tincas!

Yes that's what I was looking for, apologies for not spotting it - just testing the "Export to PDF">"Entire space" option, will report back shortly!

**Comment by jacksorjacksor** [comment]
Using the "Export to PDF">"Entire space" for me results in this error:

![image](https://github.com/umbraco/UmbracoDocs/assets/5838388/754a3a96-6a0f-4fb1-baa3-c594c9b09355)

...but at least I can start downloading the relevant pages at the very least, if required.

**Comment by alina-tincas** [comment]
No worries @jacksorjacksor! Can you please share with me the docs link that you have tried to export the space and failed? 🤔

**Comment by jacksorjacksor** [comment]
@alina-tincas yup! Here you go: https://docs.umbraco.com/umbraco-cms/v/10.latest-lts/

**Comment by alina-tincas** [comment]
Thank you! It seems to work for Deploy documentation but not for CMS documentation, so am not sure if it is due to the fact that it is CMS has a bigger content 🤔 I have reached out to Gitbook support, so I will get back to you once I get an answer from them on what could be the issue or resolution!

**Comment by jacksorjacksor** [comment]
Yeah that's what I'd assumed honestly - fab, thanks for the confirmation and thanks for looking into it!

Funnily enough this is now my second GitBook related issue after #5016 - thanks for doing what can from your side!

**Comment by alina-tincas** [comment]
No worries @jacksorjacksor always happy to help 😊 

Thank you for pointing these issues out as well! 💪🙌

**Comment by alina-tincas** [comment]
@jacksorjacksor I got some feedback from Gitbook support and they confirmed this is a limitation issue with larger spaces/pages. It is on their plan to improve this however there is no ETA when it will be resolved. 

For now, their suggestion is to try exporting PDFs in chunks of pages.

As this is not something that we can do from our side I will close this issue, but do not hesitate to create a new issue if you encounter them and then we will take it from there 👍 🙌

**Comment by jacksorjacksor** [comment]
OK thank you!
